lib/assetcompute.js

```diff
@@ -116,8 +122,13 @@ class AssetCompute {
             }
         });
         if (!response.ok) {
-            const msg = await response.text();
-            throw Error(`Unable to invoke /unregister: ${response.status}: ${msg} (details: ${JSON.stringify(response)})`);
+            const responseText = await response.text();
+            const message = `Unable to invoke /unregister: ${response.status}: ${responseText} (details: ${JSON.stringify(response)})`;
```

Comment:
Any reason why only /unregister has `(details: ${JSON.stringify(response)})` in message and not /register ?
`const message = `Unable to invoke /register: ${response.status} ${responseText}`;`

Reply:
I am honestly not sure... stringifying the entire `/process` response may be too long